cmd/scaffold.go

```diff
@@ -1050,6 +1048,8 @@ func (fnb *FlowNodeBuilder) Initialize() error {
 func (fnb *FlowNodeBuilder) RegisterDefaultAdminCommands() {
 	fnb.AdminCommand("set-log-level", func(config *NodeConfig) commands.AdminCommand {
 		return &common.SetLogLevelCommand{}
+	}).AdminCommand("set-profiler-enabled", func(config *NodeConfig) commands.AdminCommand {
+		return &common.SetProfilerEnabledCommand{}
```

Comment:
What would be the example curl command to enable the profiler on a particular node?

Is it only possible to enable the profiler, but not disable it?

Presumably this is just a way to start the profiler later when the node is running? But the interval and duration cannot be set via admin?

Reply:
yes, changing the interval and duration dynamically is a bit harder, maybe we can do that later in a separate PR